deck.gl/examples/pure-js/angular/carto2/package.json

```diff
@@ -21,14 +21,14 @@
     "@angular/platform-browser": "~11.2.0",
     "@angular/platform-browser-dynamic": "~11.2.0",
     "@angular/router": "~11.2.0",
-    "@deck.gl/carto": "8.4.9",
-    "@deck.gl/core": "8.4.9",
-    "@deck.gl/layers": "8.4.9",
+    "@deck.gl/carto": "8.5.0",
```

Comment:
We should have now 8.6.0 right?
How about using ^8.6.0 like imports? (so the people grab this when copying examples...)

Reply:
Updated to 8.6.5. Sometimes there are breaking changes in minor versions, so I prefer to pin the example to a specific version.